.eslintignore

```diff
@@ -0,0 +1 @@
+luxon.js
```

Comment:
Sr we kindly accept your request to have the json files in the repo. 
https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md
But keep in mind the following information:
![image](https://user-images.githubusercontent.com/14119855/124987101-14e46180-e002-11eb-96f0-dfc88bcb7664.png)
There are no requirements to have the .json files inside the repository since they are just needed to run the linterns locally. 
Thanks to the linters.yml they are running as Github Actions,  and as you can see there were not errors found.
So the next time you review a project , keep that in mind please.